setup-and-run.sh

```
@@ -0,0 +1,476 @@
+# Load environment variables (properly)
+GITHUB_TOKEN=""
+LINEAR_API_KEY=""
+
+if [ -f ".env" ]; then
+    while IFS='=' read -r key value; do
+        # Skip comments and empty lines
+        [[ $key =~ ^#.*$ ]] && continue
+        [[ -z $key ]] && continue
+        
+        # Remove quotes if present
+        value=$(echo "$value" | sed -e 's/^["\'"']//;s/["\'"']$//')
```

Shellcheck (0.11.0)

[error] Couldn't parse this command expansion. Fix to allow more checks.

(SC1073)